requirements.txt

```
@@ -1,4 +1,4 @@
-Pillow==6.2.0
+Pillow==7.1.0
```

```suggestion
Pillow==6.2.2
```

Try 6.2.2 to fix at least some of the vulnerabilities